src/apps/frontend/pages/login/login-form.tsx

```diff
@@ -41,7 +41,7 @@ const LoginForm: React.FC<LoginFormProps> = ({
   const navigate = useNavigate();
   useEffect(() => {
     if (currentLoginMethod === constant.PHONE_NUMBER_BASED_AUTHENTICATION) {
-      navigate(routes.SIGNUP);
+      navigate(routes.PHONE_LOGIN);
```

Comment:
This is somewhat expected.

The route is same for both phone and email.

Ideally user should not be able to reach to this form if authentication mechanism is phone.

I would expect the component mounted on route /login should be different based on auth mechanism